client/apollo/css/src/Modal/ModalApollo.scss

```
@@ -18,7 +18,11 @@
   --modal-footer-horizontal-padding: var(--modal-default-padding);
   --modal-footer-gap: calc(16 / var(--font-size-base) * 1rem);
 
-  :has(.af-modal__body-vertical-scroll) {
+  &__body:not(:has(~ .af-apollo-modal__footer)) {
+    --modal-body-padding-bottom: var(--modal-default-padding);
+  }
+
+  :has(.af-apollo-modal__body-vertical-scroll) {
```

Le `:has` fonctionnait sans `&` ?
```suggestion
  &:has(.af-apollo-modal__body-vertical-scroll) {
```

---

oui, ça fonctionne sans le &